esx_service/vmdk_ops.py

```diff
@@ -104,6 +104,10 @@
 MAX_SKIP_COUNT = 16       # max retries on VMCI Get Ops failures
 VMDK_ADAPTER_TYPE = 'busLogic'  # default adapter type
 
+#VMCI Version
+# !!! server side VMCI version, need to change according with client side version !!! 
+VMCI_VERSION = "12"
```

Comment:
@shaominchen  - the only difference I see between the current code and your suggestion is that you want to rename the `vmdk_ops.py:PROTOCOL_VERSION` to `SERVER_PROTOCOL_VERSION` (note - it is NOT the version of server. It is the version of over-VMCI protocol the server expect) .
and to rename `esx_vmdkcmd.go:protocolCurrentVersion` to `clientProtocolVersion`. 

Is it what you are saying  ?

I agree, it makes the code easier  to read and understand.